apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx

```diff
@@ -64,42 +83,40 @@ const style = {
     padding: '2em',
     textAlign: 'center',
   },
+  footer: {
+    flex: '0 0 0',
```

Comment:
I believe IE11 only has partial support for flexbox. Might be worth a look in that browser.